skillmap/src/App.tsx

```diff
@@ -138,13 +138,17 @@ class AppImpl extends React.Component<AppProps, AppState> {
             force = !!mlang && !!mlang[2];
         }
 
+        const defLocale = pxt.appTarget.appTheme.defaultLocale;
+        if (!force && !pxt.Util.isLocaleEnabled(useLang!)) {
+            useLang = defLocale;
```

Comment:
I don't get this one as well. You are setting useLang if it is not isLocaleEnabled?

Reply:
Sorry ! tripped me off. This looks good.